### ISSUE TITLE
Traceback on environment creation

### ISSUE THREAD

**Issue by wasade** [conda/conda#5574]
Relevant dump from `conda` is below. Prior to this, I executed `conda update conda` when outside of an environment. Additionally, I issued a `conda clean --all` prior to the install attempt.

```bash
$ conda create -n qiime2-2017.6 --file https://data.qiime2.org/distro/core/qiime2-2017.6-conda-linux-64.txt
# NOTE: packages downloaded removed as it is large
An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues



Current conda install:

               platform : linux-64
          conda version : 4.3.22
       conda is private : False
      conda-env version : 4.3.22
    conda-build version : 2.0.10
         python version : 3.4.5.final.0
       requests version : 2.14.2
       root environment : /home/mcdonadt/miniconda3  (writable)
    default environment : /home/mcdonadt/miniconda3
       envs directories : /home/mcdonadt/miniconda3/envs
                          /home/mcdonadt/.conda/envs
          package cache : /home/mcdonadt/miniconda3/pkgs
                          /home/mcdonadt/.conda/pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/linux-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : None
             netrc file : None
           offline mode : False
             user-agent : conda/4.3.22 requests/2.14.2 CPython/3.4.5 Linux/2.6.32-504.23.4.el6.x86_64 CentOS/6.6 glibc/2.12
                UID:GID : 1156392:550

`$ /home/mcdonadt/miniconda3/bin/conda create -n qiime2-2017.6 --file https://data.qiime2.org/distro/core/qiime2-2017.6-conda-linux-64.txt`




    Traceback (most recent call last):
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/exceptions.py", line 632, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/cli/main.py", line 137, in _main
        exit_code = args.func(args, p)
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/cli/main_create.py", line 68, in execute
        install(args, parser, 'create')
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/cli/install.py", line 187, in install
        explicit(specs, prefix, verbose=not context.quiet, index_args=index_args)
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/misc.py", line 110, in explicit
        execute_actions(actions, index, verbose=verbose)
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/plan.py", line 830, in execute_actions
        execute_instructions(plan, index, verbose)
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/instructions.py", line 247, in execute_instructions
        cmd(state, arg)
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/instructions.py", line 108, in UNLINKLINKTRANSACTION_CMD
        txn.execute()
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/core/link.py", line 263, in execute
        self.verify()
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/core/link.py", line 251, in verify
        self.num_unlink_pkgs),
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/core/link.py", line 248, in <genexpr>
        exceptions = tuple(exc for exc in concatv(
      File "/home/mcdonadt/miniconda3/lib/python3.4/site-packages/conda/core/link.py", line 192, in _verify_individual_level
        log.debug(format_exc())
      File "/home/mcdonadt/miniconda3/lib/python3.4/traceback.py", line 256, in format_exc
        return "".join(format_exception(*sys.exc_info(), limit=limit, chain=chain))
      File "/home/mcdonadt/miniconda3/lib/python3.4/traceback.py", line 181, in format_exception
        return list(_format_exception_iter(etype, value, tb, limit, chain))
      File "/home/mcdonadt/miniconda3/lib/python3.4/traceback.py", line 146, in _format_exception_iter
        for value, tb in values:
      File "/home/mcdonadt/miniconda3/lib/python3.4/traceback.py", line 125, in _iter_chain
        context = exc.__context__
    AttributeError: 'NoneType' object has no attribute '__context__'
```

**Comment by nehaljwani** [comment]
Probably related https://github.com/conda/conda/issues/4519

**Comment by nehaljwani** [comment]
@wasade Could you please type the same command with `-vvv` as argument and share the output?

**Comment by wasade** [comment]
Thanks for the quick response @nehaljwani! I hit the comment length limit for github so I dropped it [here](https://pastebin.com/6V4KVEeT)

**Comment by kalefranz** [comment]
believe this can be closed following solution for #4519
